app/controllers/stories_controller.rb

```diff
@@ -122,10 +125,11 @@ def handle_tag_index
     @num_published_articles = if @tag_model.requires_approval?
                                 Article.published.cached_tagged_by_approval_with(@tag).size
                               else
-                                Article.published.cached_tagged_with(@tag).size
+                                Article.published.cached_tagged_with(@tag).where("score > 2").size
                               end
-    number_of_articles = user_signed_in? ? 5 : 45
-    @stories = Articles::Feed.new(number_of_articles: number_of_articles, tag: @tag).published_articles_by_tag
+    @number_of_articles = user_signed_in? ? 5 : SIGNED_OUT_RECORD_COUNT
```

Comment:
I'm assuming we intentionally want to increment this from 45 to 60. Should we include a comment with `SIGNED_OUT_RECORD_COUNT` to explain how/where we're getting those numbers? They feel arbitrary to me at the moment, but I just might not have enough context for this.